.env.example

```diff
@@ -11,3 +11,4 @@ EMAIL_URL=dummymail://
 DEFAULT_FROM_EMAIL=webmaster@localhost
 SERVER_EMAIL=root@localhost
 ADMINS=Root User <root@localhost.dev>
+# VAPID_PRIVATE_KEY_PATH=data/private_key.pem # push notifications, create in `data/` with `vapid --gen`
```

Comment:
placeholder comment: remember to add the issues from the PR description

Reply:
the last one already exists (#577)